atc/worker/gardenruntime/worker.go

```diff
@@ -217,6 +218,7 @@ func (worker *Worker) createGardenContainer(
 			Properties: garden.Properties{
 				userPropertyName: fetchedImage.Metadata.User,
 			},
+			NetOut: worker.getNetOut(containerSpec.Hermetic),
```

Comment:
Can we only set `NetOut` when `hermetic` is true? And do nothing when it's false?

Reply:
Yes thx for the review.